openssl-builder/Dockerfile

```diff
@@ -46,4 +46,6 @@ COPY openssl ./
 RUN equivs-build openssl
 COPY libssl ./
 RUN equivs-build libssl
-RUN apt-get install ca-certificates
+COPY ca-certificates ./
+RUN equivs-build ca-certificates
+RUN apt-get install -y ca-certificates
```

Comment:
Is it needed step on the builder image?

Reply:
yes we are copying this package artifacts to `phusion-ruby-fips` and `ubuntu-ruby-fips `images